pypit/arparse.py

```diff
@@ -2283,6 +2295,55 @@ def bias_number(self, v):
         #key_min_val(v,-1)
         self.update(v)
 
+    def dark_canbe(self, v):
+        """ If there are frames that will be a bias in addition to other frame types,
```

Comment:
Should this description say "dark" instead of "bias"?

Same comment for the following 3.